libs/core/state/src/states/state.enum.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * The DaffState enumerates the vast majority of state modifications that can exist 
+ * in UI applications. 
+ */
+export enum DaffStates {
```

Comment:
Is it DaffState singular or DaffStates plural?

Reply:
DaffState singular is better. Changed.